runhouse/rns/rns_client.py

```diff
@@ -630,3 +632,33 @@ def contents(self, name_or_path, full_paths):
         return folder(name=name_or_path, path=folder_url).resources(
             full_paths=full_paths
         )
+
+    async def asave_status_metrics_to_den(
```

Comment:
wdyt about putting this in the cluster servelet instead?

Reply:
hmmm, yeah at first it made sense to me to put this in the rns_client, but on second thought it is indeed more appropriate the have it living inside the cluster servlet.